### PR TITLE
Jekyll integration and HIP-1 updates

### PR DESCRIPTION
**Pull request by sergmetelin**

### CODE REVIEW

File: HIP/hip-1.md [hunk 1]
```diff
@@ -74,11 +75,15 @@ Once the champion has discovered with the Hedera community the acceptability of
 
 The standard HIP workflow is:
 
-* You, the HIP author, fork the HIP repository, and create a file named hip-0000-my-feature.md (where “my-feature” is descriptive) that contains your new HIP. Use 0000 as your draft HIP number. This file should be created from the HIP template.
+* You, the HIP author, fork the HIP repository, and create a file named `hip-0000-my-feature.md` (where "my-feature" is descriptive) that contains your new HIP. Use 0000 as your draft HIP number. This file should be created from the HIP template.
+
+* In the "Type" header field, enter "Standards Track", "Informational", or "Process" as appropriate, and for "Status" field enter "Draft".
```
Comment (SimiHunjan) [inline HIP/hip-1.md:80]: Did we want the status to be "WIP" first and when merged HIP editors update to  "Draft"

File: HIP/hip-1.md [hunk 2]
```diff
@@ -74,11 +75,15 @@ Once the champion has discovered with the Hedera community the acceptability of
...
 
-* In the “Type” header field, enter “Standards Track”, “Informational”, or “Process” as appropriate, and for “Status” field enter “Draft”.
+* Push this to your Github fork and submit a draft pull request ([https://github.blog/2019-02-14-introducing-draft-pull-requests/](https://github.blog/2019-02-14-introducing-draft-pull-requests/)).
 
-* Push this to your Github fork and submit a pull request.
+* As you iterate through the community feedback, you add changes to the HIP which will be reflected in the draft pull request.
+
+* Once you finalize the changes related to the feedback, convert the pull request from "Draft" to "Review". This will signal an editor to review your HIP.
```
Comment (SimiHunjan) [inline HIP/hip-1.md:86]: We should add note about the DCO requirements before it is submitted for review.
Add some reference links: https://www.secondstate.io/articles/dco/, https://github.com/apps/dco

File: HIP/hip-1.md [hunk 3]
```diff
@@ -90,7 +95,7 @@ The standard HIP workflow is:
 
     Editors are generally quite lenient about this initial review, expecting that problems will be corrected by the reviewing process.
 
-    If the HIP isn’t ready for approval, an editor will send it back to the author for revision, with specific instructions.
+    If the HIP isn’t ready for approval, an editor will send it back to the author for revision, with specific instructions and will change the status of the HIP back to the "Draft".
 
 * Once approved, editors will assign your HIP a number and merge your PR into the main branch.
```
Comment (SimiHunjan) [inline HIP/hip-1.md:101]: What status are we updating the HIP to once it is reviewed and merged? I think before it is merged the status is "Review." Is it the same status after the merge?

File: HIP/hip-1.md [hunk 4]
```diff
@@ -104,31 +109,31 @@ HIP authors are responsible for collecting community feedback on a HIP before su
...
-
-To allow gathering of additional design and interface feedback before committing to long term stability, a HIP may also be marked as “Provisional”. This is short for “Provisionally Accepted”, and indicates that the proposal has been accepted for inclusion, but additional user feedback is needed before the full design can be considered “Final”. Unlike regular accepted HIPs, provisionally accepted HIPs may still be Rejected or Withdrawn even after the related changes have been implemented.
+To allow gathering of additional design and interface feedback before committing to long term stability, a HIP may will be marked as "Last Call". This indicates that the proposal has been accepted for inclusion, but additional user feedback is needed before the full design can be considered "Accepted". A HIP in the "Last Call" status may still be Rejected or Withdrawn even after the related changes have been implemented.
```
Comment (SimiHunjan) [inline HIP/hip-1.md:118]: We should add how long HIPs will have for "Last Call" status at minimum.

File: HIP/hip-1.md [hunk 5]
```diff
@@ -104,31 +109,31 @@ HIP authors are responsible for collecting community feedback on a HIP before su
...
 
-Wherever possible, it is considered preferable to reduce the scope of a proposal to avoid the need to rely on the “Provisional” status, as this status can lead to version compatibility challenges in the wider Hedera ecosystem.
+Once a HIP has been accepted, the reference implementation must be completed. When the reference implementation is complete and incorporated into the main source code repository, the status will be changed to "Final". A Standards Track HIP must include a specification and a reference implementation in order to be considered for a "Final" status. For Process HIPs and Standards Track HIPs that introduce protocol standards, once the process or protocol has been implemented and recognized as official by a combination of the community, core developers, and the Council, the status will be changed to "Final".
```
Comment (SimiHunjan) [inline HIP/hip-1.md:120]: How do we define an accepted HIP?

File: HIP/hip-1.md [hunk 6]
```diff
@@ -41,15 +42,15 @@ There are three kinds of HIP:
...
 
-Hedera Core Developers or “core developers” include those who are tasked with development of any part of the Hedera platform or ecosystem; this includes employees of Hedera, contractors hired by Hedera directly, and community developers who have received grants to develop and maintain a project.
+Hedera Core Developers or "core developers" include those who are tasked with development of any part of the Hedera platform or ecosystem; this includes employees of Hedera, contractors hired by Hedera directly, and community developers who have received grants to develop and maintain a project.
 
 ### HIP Editors
 
-The HIP editors or “editors” are individuals responsible for managing the administrative and editorial aspects of the HIP workflow. HIP editorship is by invitation of the current editors or by assignment by the Council.
+The HIP editors or "editors" are individuals responsible for managing the administrative and editorial aspects of the HIP workflow. HIP editorship is by invitation of the current editors or by assignment by the Council.
 
 ### Start with an idea for Hedera
```
Comment (SimiHunjan) [inline HIP/hip-1.md:55]: Should we add a section that summarizes all of the status and what it means?

**Review by SimiHunjan — CHANGES_REQUESTED** [review]

File: HIP/hip-1.md (hunk 1 above)
Comment (sergmetelin) [reply HIP/hip-1.md:80]: Good call, updating.

File: HIP/hip-1.md (hunk 2 above)
Comment (sergmetelin) [reply HIP/hip-1.md:86]: Noted, updated.

File: HIP/hip-1.md (hunk 3 above)
Comment (sergmetelin) [reply HIP/hip-1.md:101]: It will be "WIP".
"Draft" is assigned only after a successful review by the editor.
Updated.

File: HIP/hip-1.md (hunk 4 above)
Comment (sergmetelin) [reply HIP/hip-1.md:118]: Added

File: HIP/hip-1.md (hunk 5 above)
Comment (sergmetelin) [reply HIP/hip-1.md:120]: An accepted HIP is a HIP that went through the "Last Call" period without changes and its status has been converted to "Accepted" at the end of `last-call-date-time` period. 

Update in the HIP text to reflect this.

File: HIP/hip-1.md (hunk 1 above)
Comment (sergmetelin) [reply HIP/hip-1.md]: Actually, deprecated "WIP" status in favour of "Draft" to simplify the process

File: HIP/hip-1.md (hunk 6 above)
Comment (sergmetelin) [reply HIP/hip-1.md:55]: Done

File: HIP/hip-1.md [hunk 7]
```diff
@@ -152,41 +190,46 @@ Each HIP should have the following parts/sections:
...
 
-Each HIP must begin with a header preamble in list format. The headers must appear in the following order. Headers marked with “*” are optional and are described below. All other headers are required.
+Each HIP must begin with a header preamble in a table format. The headers must appear in the following order. Headers marked with "*" are optional and are described below. All other headers are required.
 
-- hip: HIP number (this is determined by the HIP editor)
+- hip: HIP number (this is determined by the PR number and set by the editor)
 - title: HIP title
-- author: a list of the author’s or authors’ name(s) and/or username(s), or name(s) and email(s).
+- author: a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s).
+- \* working-group: a list of the technical and business stakeholders' name(s) and/or username(s), or name(s) and email(s).
```
Comment (SimiHunjan) [inline HIP/hip-1.md:226]: There is an extra `\*` here

File: HIP/hip-1.md [hunk 8]
```diff
@@ -152,41 +190,46 @@ Each HIP should have the following parts/sections:
...
 - type: <Standards Track | Informational | Process>
 - \* category: <Core | Service | API | Mirror | Application>
```
Comment (SimiHunjan) [inline HIP/hip-1.md:228]: There is an extra `\*` here

File: HIP/hip-1.md [hunk 9]
```diff
@@ -152,41 +190,46 @@ Each HIP should have the following parts/sections:
...
+- needs-council-approval: <true | false>
 - status: <Draft | Active | Inactive | Provisional | Deferred | Rejected | Withdrawn | Final | Replaced >
 - created: date created on
+- last-call-date-time: the anticipated date and time when this HIP will change status to `Last Call`, filled out by the editor
 - \* discussions-to: a URL pointing to the official discussion thread
```
Comment (SimiHunjan) [inline HIP/hip-1.md:233]: There is an extra `\*` here

File: HIP/hip-1.md [hunk 10]
```diff
@@ -152,41 +190,46 @@ Each HIP should have the following parts/sections:
...
 - \* updated: comma separated list of dates
```
Comment (SimiHunjan) [inline HIP/hip-1.md:234]: There is an extra `\*` here

File: HIP/hip-1.md [hunk 11]
```diff
@@ -152,41 +190,46 @@ Each HIP should have the following parts/sections:
...
 - \* requires: HIP number(s)
```
Comment (SimiHunjan) [inline HIP/hip-1.md:235]: There is an extra `\*` here

**Review by SimiHunjan — CHANGES_REQUESTED** [review]

**Review by SimiHunjan — DISMISSED** [review]

**Review by SimiHunjan — APPROVED** [review]

**Review by SimiHunjan — APPROVED** [review]

**Pull request merged**